tests/unittests/PartitionerTest.cpp

```diff
@@ -615,6 +618,9 @@ TEST_F(PartitionerTest, SimpleHeterogeneousPartitioning) {
 /// Heterogeneous Partition. In this test, "Mul" is not supported in
 /// Interpreter backend, and "Sub" is not supported in CPU backend.
 TEST_F(PartitionerTest, heterogeneousPartitioningWithNonSupportedNodes) {
+#ifndef GLOW_WITH_CPU
```

Comment:
@beicy it seems that PartitionerTest implicitly assumes that Glow is built with the CPU support.
Is there any good way to make these tests more robust?